include/wil/common.h

```diff
@@ -613,10 +613,10 @@ namespace wil
     }
 
     template <>
-    _Post_satisfies_(return == !!val)
+    _Post_satisfies_(return == (val != 0))
     __forceinline constexpr bool verify_bool<unsigned char>(unsigned char val)
     {
-        return !!val;
+        return (val != 0);

```

Comment:
@ChrisGuzak do you recall if there's any historical reason we've preferred `!!` over `!= 0`? Memory seems to suggest that there's code-gen differences, but I'm not sure if I'm confusing that in with `= { 0 }` vs `= {}`